### DISCUSSION TITLE
[GS] ParseMessage Code ← World를 1.3.0으로 바꿨더니 왼쪽의 로그가 계속 나옵니다.

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#707] [5. Q&A]
![image](https://user-images.githubusercontent.com/5895396/167076325-291276e1-8221-4e04-bcd6-18009f00ad7d.png)

- 1.3.0으로 Zepeto.World를 갱신한 후로 위의 내용 같은 로그가 계속 나오는데, 이건 어떤 로그인가요?
>> index에 보내는 메시지를 Parsing 할 때마다 나오는 로그일까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 최신 버전 1.4.0에서도 계속 나타나는지 확인 부탁 드립니다.

**Reply by ryusiin** [reply]
수정 확인했습니다:>
감사합니다!
